### PR TITLE
Add blog post on Gaia-x Hackathon nr. 5

### PR DESCRIPTION
**Pull request by itrich**
Signed-off-by: Eduard Itrich <eduard@itrich.net>

### CODE REVIEW

**Comment by itrich** [comment]
Dear participants @fkr, @matfechner, @JuanPTM, @frosty-geek: Feel free to ammed this blog post and add yourself to the list of authors. Thank you for being around the last two days. It was great to work with you! :rocket:

File: _i18n/en/_posts/blog/2022-09-28-gaia-x-hackathon-5.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+## Day 2
+
+The second day started with a very good discussion about user federation between
+multiple SCS-compliant cloud offerings. We identified three possible scenarios
+for *Jane Doe*:
+
+* CSP Alpha (with OpenStack + Keycloak Alpha)
+* CSP Beta (with OpenStack + Keycloak Beta)
+* Company ACME Org. with user Jane Doe
+  * without any IAM (but with a SSI kept in a wallet)
```
Comment (tibeer) [inline _i18n/en/_posts/blog/2022-09-28-gaia-x-hackathon-5.md:103]: ... with **an** SSI ...

File: _i18n/en/_posts/blog/2022-09-28-gaia-x-hackathon-5.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+      </div>
+    {% endif %}
+  {% endfor %}
+</div>
+
+## Open questions
+
+We identified several challenges that are still open to discussion.
+
+* Jane Doe needs so sign in to have an entity on Keycloak Alpha and Keycloak Beta. How will rights and privileges then be mapped by Keycloak Alpha and Beta without knowing anything about Jane Doe (besides her company)?
```
Comment (tibeer) [inline _i18n/en/_posts/blog/2022-09-28-gaia-x-hackathon-5.md:147]: How will permissions and privileges ...

File: _i18n/en/_posts/blog/2022-09-28-gaia-x-hackathon-5.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+[an own proposal](https://gitlab.com/gaia-x/gaia-x-community/gx-hackathon/gx-hackathon-5/-/wikis/Hackathon-5-Proposals#07-cloud-federation-via-oidc). The goal of our track was to realize user federation at the infrastructure level
+with Keycloak via OpenID Connect.
+
+<figure class="figure mx-auto d-block" style="width:50%">
+    {% asset 'blog/gx-hackathon-5.jpg' class="figure-img w-100" %}
+</figure>
+
+Following the organizers' recommendation, we joined forces with the team from [walt.id](https://walt.id),
+which in the end proved to be a great opportunity and resulted in an interesting use case.
+After aligning our both proposals, we concluded to work together on realizing a
```
Comment (tibeer) [inline _i18n/en/_posts/blog/2022-09-28-gaia-x-hackathon-5.md:23]: on realizing **an** ...

File: _i18n/en/_posts/blog/2022-09-28-gaia-x-hackathon-5.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+  {% endfor %}
+</div>
+
+## Open questions
+
+We identified several challenges that are still open to discussion.
+
+* Jane Doe needs so sign in to have an entity on Keycloak Alpha and Keycloak Beta. How will rights and privileges then be mapped by Keycloak Alpha and Beta without knowing anything about Jane Doe (besides her company)?
+* CSP have to setup own Keycloaks that have to accept everything from the third party Keycloak provider. Will this find acceptance?
+* We need to tackle how permission rights for Jane Doe are maintained – especially across multiple CSP.
```
Comment (tibeer) [inline _i18n/en/_posts/blog/2022-09-28-gaia-x-hackathon-5.md:149]: ~~permission rights~~ permissions

**Review by tibeer — COMMENTED** [review]
Awesome work @itrich! just a few minor things I've found.

**Comment by matfechner** [comment]
LGTM

**Comment by itrich** [comment]
Hi @philpotisk, @severinstampler, @waltkb,

as promised we've summarized our achievements during Hackathon Gaia-X nr. 5 and covered some parts about walt.id. Any feedback is highly appreciated! Hope that adding your architecture figure is okay for you.

**Pull request merged**